roles/autotested/templates/control.ini.j2

```diff
@@ -17,13 +17,17 @@ include =
 exclude = example,subexample,pretest_example,intratest_example,
           posttest_example,
           docker_cli/run_volumes,
-{% if not is_atomic %}
+{% if is_atomic %}
+          # Atomic host does not have git
           docker_cli/build/git_path,
-    {%- if is_enterprise %}
+
+    {%- if not is_enterprise %}
```

Comment:
Struggling, but IIRC, there was something about the test that doesn't work on Atomic.  Nope, it's gone, don't remember.  I can try and enable it and we can see what happens...